tools/rsdl/csdl-xml/lib/xmlSerializer.js

```diff
@@ -0,0 +1,653 @@
+var builder = require('xmlbuilder');
+
+function serializeToXml(jsonCsdl, pretty = true) 
+{
+  var entityContainerName = jsonCsdl.$EntityContainer;
+  var namespace = entityContainerName.substring(entityContainerName, entityContainerName.lastIndexOf('.'));
+
+  // Write Edmx wrapper
+  // <edmx:Edmx xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx" Version="4.0">
+  //   <edmx:DataServices>
+
+  var xmlElement = builder.begin().dec()
+    .ele('edmx:Edmx')
+      .att('xmlns:edmx','http://docs.oasis-open.org/odata/ns/edmx')
+      .att('Version','4.0');
+
+  // Write References
+  if(jsonCsdl.$References != undefined)
+  {
+    writeReferences(xmlElement, jsonCsdl.$References);
+  }
+
+  // Write Schema Element
+  //     <Schema xmlns="http://docs.oasis-open.org/odata/ns/edm" Namespace="Jetsons">
+  var dataServiceElement = xmlElement.ele('Schema');
+  var schema = dataServiceElement
+    .att('xmlns','http://docs.oasis-open.org/odata/ns/edm')
+    .att('Namespace', namespace);
+
+    var model = jsonCsdl[namespace];
+  // write model elements
+  Object.keys(model).forEach((key)=>
+  {
+    var modelElement = model[key];
+    if (Array.isArray(modelElement))
+    {
+      handleOperation(schema, key, modelElement);
+    }
+    else if (key == '$Alias')
+    {
+      schema.att("Alias", modelElement);
+    }
+    else if (key=='$Annotations')
+    {
+      writeAnnotationsElement(schema, modelElement);
+    }
+    else 
+    {
+      switch (modelElement.$Kind)
+      {
+        case 'EntityType':
+          writeEntityType(schema, key, modelElement);
+          break;
+        case 'ComplexType':
+          writeComplexType(schema, key, modelElement);
+          break;
+        case 'EnumType':
+          writeEnumType(schema, key, modelElement);
+          break;
+        case 'TypeDefinition':
+          writeTypeDefinition(schema, key, modelElement);
+          break;
+        case 'EntityContainer':
+          writeEntityContainer(schema, key, modelElement);
+          break;
+        default:
+          console.log("Unexpected Schema Element" + key);
```

Comment:
```suggestion
          console.log(`Unexpected Schema Element ${key}`);
```
Use string templates?